### ISSUE TITLE
Building Novel throws AttributeError

### ISSUE THREAD

**Issue by chokerguy** [vkbo/novelWriter#761]
When trying to build my novel in any format, I get an AttributeError. I assume this might be because I changed the working title and/or moved the folder with all the project files, but I'm not sure. Below, you can find the console output:

```Environment:
novelWriter Version: 1.2.3
Host OS: win32 (10.0.19042)
Python: 3.8.1 (0x30801f0)
Qt: 5.15.2, PyQt: 5.15.4

AttributeError:
'Config' object has no attribute 'homePath'

Traceback:
  File "C:\Program Files\novelWriter 1.2.3\nw\gui\build.py", line 434, in <lambda>
    self.saveODT.triggered.connect(lambda: self._saveDocument(self.FMT_ODT))

  File "C:\Program Files\novelWriter 1.2.3\nw\gui\build.py", line 846, in _saveDocument
    saveDir = self.mainConf.homePath

**Comment by vkbo** [comment]
That is indeed a bug. This is a corner case issue that apparently isn't covered by tests. When you export a file, it will open the file dialog at the last path you used. However, if that folder no longer exists, it tries to fall back to using your home folder. But the variable that should hold the path to the home folder is missing. (There have been a few such bugs in `build.py` because the global search function in my code editor has excluded that file from searches because it contains the word "build".)

I'll make a fix. You can get around it by opening your config file and delete the variable "lastpath" that points to the non-existent folder. The config file should be in "C:/Users/<USER>/AppData/Local/novelwriter/novelwriter.conf" where "<USER>" is your username on Windows.

**Comment by chokerguy** [comment]
Thank you for the quick response! Deleting the lastpath variable entirely caused the same error. Changing the variable to a valid path worked out though. Thanks again!

**Comment by vkbo** [comment]
Interesting, because the intention is that the `lastpath` variable should default to your home directory when it isn't defined. I'll look into that as well.

**Comment by vkbo** [comment]
Hmm, did you delete the whole line in the config, or just the value? I mean the whole line, which means it will fall back to the default value which is your home folder. If you delete just the value, it sets it to an empty string, which also isn't a folder.

I tested deleting the line, which worked as I expected. Just want to confirm that there isn't another bug here.

**Comment by chokerguy** [comment]
Tried it again, you're right. I did try deleting the whole line, but I might've forgot to save right after. Now that works fine and does default to the home folder. Sorry about that!

**Comment by vkbo** [comment]
No problem. Thanks!

I also noticed you were exporting to ODT. This is a pretty new feature and I wrote the whole exporter myself, which wasn't trivial. I'm happy to take suggestions on how to improve the output. (and of course any bugs)

**Comment by chokerguy** [comment]
Damn, that's impressive. It's been going very smoothly so far. If I spot something odd, I'll be sure to report it.

**Comment by vkbo** [comment]
Bug has been fixed in the `main` branch. The fix will be included in release 1.3 that I plan to make very soon. Hopefully this weekend.
